components/esp-zigbee-console/src/cli_cmd_bdb.c

```diff
@@ -939,6 +1034,11 @@ DECLARE_ESP_ZB_CLI_CMD_WITH_SUB(network, "Network configuration",
     ESP_ZB_CLI_SUBCMD(scan,     cli_nwk_scan,     "Scan for network"),
     ESP_ZB_CLI_SUBCMD(ed_scan,  cli_nwk_ed_scan,  "Scan for energy detect on channels"),
 );
+DECLARE_ESP_ZB_CLI_CMD_WITH_SUB(linkkey, "Link Key Configuration",
+    ESP_ZB_CLI_SUBCMD(add,    cli_linkkey_add,     "Add default security link key"),
+    ESP_ZB_CLI_SUBCMD(remove, cli_linkkey_remove,  "Remove default security link key"),
+    ESP_ZB_CLI_SUBCMD(set,    cli_linkkey_set,     "Set default security link key"),
+);
```

Comment:
Conceptually, the commands here are managing Global Link Key. 

Our stack has a default one for both TC and Joining device, and multiple ones for Joining device.

```suggestion
DECLARE_ESP_ZB_CLI_CMD_WITH_SUB(linkkey, "Link Key Configuration",
    ESP_ZB_CLI_SUBCMD(add,    cli_linkkey_add,     "Add additional global link key"),
    ESP_ZB_CLI_SUBCMD(remove, cli_linkkey_remove,  "Remove additional global link key"),
    ESP_ZB_CLI_SUBCMD(set,    cli_linkkey_set,     "Set default global link key"),
);
```